libraries/standard/mqtt/src/iot_mqtt_operation.c

```diff
@@ -270,9 +270,9 @@ static bool _scheduleNextRetry( _mqttOperation_t * pOperation )
             pOperation->u.operation.periodic.retry.nextPeriodMs = IOT_MQTT_RETRY_MS_CEILING;
         }
 
-        /* In some implementations IotLog() maps to C standard printing API 
-         * that need specific primitive types for format specifiers. Also 
-         * inttypes.h may not be available on some C99 compilers, despite 
+        /* In some implementations IotLog() maps to C standard printing API
+         * that need specific primitive types for format specifiers. Also
+         * inttypes.h may not be available on some C99 compilers, despite
```

Comment:
Could these debug output lines be rewritten so that the MISRA exception isn't required?

Reply:
That change was part of another PR and we agreed upon this exception for log messages only. We need to have this due to format specifiers in log messages. For example, uint32_t may not map to unsigned long in some platforms.